var/spack/repos/builtin/packages/py-importlib-re/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyImportlibRe(PythonPackage):
```

Comment:
```suggestion
class PyImportlibResources(PythonPackage):
```
Need to rename `package.py` directory too.